js/components/videoQualityPanel.js

```diff
@@ -33,43 +54,67 @@ var VideoQualityPanel = React.createClass({
       'oo-selected': this.state.selected == 'auto'
     });
 
-    //add auto btn to beginning of array
-    bitrateButtons.unshift(
-      <li className="oo-auto-li" key='auto-li'>
-        <a className={autoQualityBtn} key='auto' onClick={this.handleVideoQualityClick.bind(this, 'auto')}>
-          <div className="oo-quality-auto-icon">
-            <Icon {...this.props} icon="auto"/>
-          </div>
-          <div className="oo-quality-auto-label">Auto</div>
-        </a>
-      </li>
-    );
+   if (autoQualityBtn === "oo-quality-auto-btn oo-selected") {
```

Comment:
please simplify this conditional statement, there is no need to duplicate logic.

See comment in fontSizeTab.js for more details.